engine/schema/src/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -51,11 +51,11 @@
 import com.cloud.upgrade.dao.Upgrade440to441;
 import com.cloud.upgrade.dao.Upgrade441to442;
 import com.cloud.upgrade.dao.Upgrade442to450;
-import com.cloud.upgrade.dao.Upgrade443to450;
+import com.cloud.upgrade.dao.Upgrade443to444;
```

Comment:
I understand the concern, but it's not an altered upgrade path per se, rather than the previous logical hierarchy was messed up and now it's in correct logical order.

`443to450` is replaced by `443to444` and the hierarchy is:
- `443to444` (Empty `schema-443to444.sql`. There's no `schema-443to444-cleanup.sql`) 
- `444to450` (Doesn't have anything in it and it extends `Upgrade442to450`)
- `443to450` itself is empty and only extends `Upgrade442to450` which hierarchy is already aware of.

And `452to460` is replaced by `452to453` and the hierarchy is:
- `452to453` (Empty `schema-452to453.sql` and `schema-452to453-cleanup.sql`) 
- `453to460` (Doesn't have anything in it and it extends `Upgrade452to460`)

Reply:
I did two tests and start management server to see if normal stuff works: 
- Install on brand new database
- Upgrade to 4.12 from 4.10